main.py

```
@@ -426,12 +427,22 @@ async def generate_progress():
     )
 
 
+def delete_file_after_download(path: Path):
+    try:
+        if path.exists():
+            os.remove(path)
+            print(f"[DEBUG] Deleted file after download: {path}")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)